step-capstone/src/components/TripList.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Grid } from '@material-ui/core'
+import TripItemComponent from "./TripItemComponent"
+import Trip from "./Trip"
+import {
+  Switch,
+  Route,
+  Redirect
+} from "react-router-dom";
+
+class TripList extends React.Component{
+
+  constructor(props){
+    super(props);
+    this.state = {selectedTrip : null};
+    this.handleOpenTrip = this.handleOpenTrip.bind(this);
+  }
+
+  getTrips(userId) {
+    //Test Data
+    //TODO: Replace test data with call to firestore method
+    const tripTestOne = {
+      id : 1,
+      title : "London trip", 
+      startDate : new Date (2020, 8, 10),
+      endDate : new Date (2020, 8, 15),
+      description : "lorem ipsum ", 
+    }
+    const tripTestTwo = {
+      id : 2,
+      title : "Spain trip", 
+      startDate : new Date (2020, 12, 10),
+      endDate : new Date (2020, 12, 15),
+      description : "lorem ipsum ", 
+    }
+    return [tripTestOne, tripTestTwo];
```

Comment:
Would be slightly cleaner to put the test data in a unified place, then import it into here. That way it's easy to do things like add test users.

Reply:
definitely agree. when we move on to adding explicit testing i'll do something like that. this stuff actually gets deleted in my other PR  once firestore is established so I think for now it is fine to leave it.